waiter/src/waiter/service_description.clj

```diff
@@ -478,7 +485,8 @@
    parameters and metadata) is provided.
    The token-description consists of the following keys: :service-description-template and :token-metadata"
   [config]
-  {:service-description-template (select-keys config service-description-keys)
+  {:request-parameters (select-keys config request-parameter-keys)
```

Comment:
`fallback-period-secs` is only a `request-parameter` when it's a parameter of a request.  `fallback-period-secs` most appropriately fits into `token-metadata`.  If we must have a 3rd map, let's give it a better name.

Reply:
It is different from system generated metadata, something the user can specify. Happy to rename the map. Any suggestions? `user-metadata-keys`?

Reply:
I can live with `user-metadata-keys`.  Should we name the other map `system-metadata-keys`?

Reply:
Okay, done. We now have `user-metadata-keys` and `system-metadata-keys` that form `token-metadata-keys`